vslib/src/sai_vs_generic_get.cpp

```diff
@@ -107,7 +107,7 @@ sai_status_t internal_vs_generic_get(
 
             if (status != SAI_STATUS_SUCCESS)
             {
-                SWSS_LOG_ERROR("%s read only not implemented on %s",
+                SWSS_LOG_DEBUG("%s read only not implemented on %s",
```

Comment:
I think it is better to be INFO level, so we can change level to catch this but not mixed with DEBUG which is too verbose.

Reply:
Sure